### DISCUSSION TITLE
How to install TRUST on a Cluster?

### DISCUSSION THREAD

**Discussion by Mrluo123** [cea-trust-platform/trust-code#2] [General]
Hello,
As we see, there are many libries needed when instaling TRUST.
But it's not easy to install these libaries in a High Performance Cluster beacuse it's impossible to obtain the permission.

So, is there any other method to install TRUST on HPC?

SIncerely,
Xiao

**Comment by pledac** [comment]
Hello,

What are the libraries missing on the cluster for TRUST? There is no other method for the moment to build the sources on the cluster thanks to the provided libraries (externalpackages) and a MPI library on the cluster.

Thanks

**Reply by Mrluo123** [reply]
Hi,
Thanks for your reply and sorry for my delay.

Well, currently I am running TRUST on my personal computer but in the near future, we consider making it run on a high-performance cluster because of the calculation burden. 

When I install TRUST on my personal computer, I encounted some problems but they are all solved with some efforts and time. But as I see, there are so many libraries needed by TRUST, so I thought it may be difficult to install it on HPC.

So my question is actually about the experience to install TRUST on an HPC.  Have you already got any experience in installing it on the cluster?

Thanks.
Regards,
Xiao

**Reply by pledac** [reply]
Hello,

Yes we have TRUST currently installed on a lot of clusters. Do you remember the missing libraries during your attempts on your PC ?

Thanks
